lib/ansible/plugins/action/async.py

```diff
@@ -89,7 +89,15 @@ def run(self, tmp=None, task_vars=None):
         async_limit = self._task.async
         async_jid   = str(random.randint(0, 999999999999))
 
-        async_cmd = [env_string, remote_async_module_path, async_jid, async_limit, remote_module_path]
+        # call the interpreter for async_wrapper directly
+        # this permits use of a script for an interpreter on non-Linux platforms
+        # TODO: re-implement async_wrapper as a regular module to avoid this special case
+        interpreter = shebang.replace('#!', '')
```

Comment:
same thing here- a strip() for leading spaces in the shebang might be a good idea- less an issue here since it's our code...